chart/compass/values.yaml

```diff
@@ -1114,6 +1120,12 @@ global:
       providerAppClientID: "provider-client-id"
       providerAppTenantID: "provider-app-tenant-id"
       apiName: "Test API Name"
+  subscription:
```

Comment:
I think this fits more in the current `tests.subscription` properties. Because this new one is the third subscription property(one for the director, in the tests, and now the new one) and it's a little bit harder to follow where/what is used. And these configurations mostly are used in the tests(+ one of them in the ext-svc-mock which is also "part of the tests/mocks")